docs/projects/ad9434_fmc/index.rst

```diff
@@ -0,0 +1,225 @@
+:adi:`AD9517-4`.
+
+CPU/Memory interconnects addresses
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The addresses are dependent on the architecture of the FPGA, having an offset
+added to the base address from HDL(see more at :ref:`architecture`).
+
+==================== ===============
+Instance             Zynq/Microblaze
+==================== ===============
+axi_ad9434           0x44A0_0000    
+axi_ad9434_dma       0x44A3_0000    
+==================== ===============
+
+SPI connections
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :widths: 25 25 25 25
+   :header-rows: 1
+
+   * - SPI type
+     - SPI manager instance
+     - SPI subordinate
+     - CS
+   * - PS
+     - SPI 0
+     - AD9517
+     - 1
+   * - PS
+     - SPI 0
+     - AD9434BCPZ
+     - 0
+
+Interrupts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Below are the Programmable Logic interrupts used in this project.
+
+================ === ========== ===========
+Instance name    HDL Linux Zynq Actual Zynq
+================ === ========== ===========
+axi_ad9783_dma   13  57         89             
+================ === ========== ===========
+
+These are the board-specific interrupts 
+(found in :git-hdl:`here <projects/ad9434_fmc/common/ad9434_bd.tcl>`).
+
+Building the HDL project
+-------------------------------------------------------------------------------
+
+The design is built upon ADI's generic HDL reference design framework.
+ADI does not distribute the bit/elf files of these projects so they
+must be built from the sources available :git-hdl:`here </>`. To get
+the source you must
+`clone <https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository>`__
+the HDL repository.
+
+Then go to the project location (**projects/ad9434_fmc/carrier**) and run the 
+make command by typing in your command prompt (this example is for 
+:xilinx:`ZC706`):
+
+**Linux/Cygwin/WSL**
+
+.. code-block::
+
+   user@analog:~$ cd hdl/projects/ad9434_fmc/zc706
+   user@analog:~/hdl/projects/ad9434_fmc/zc706$ make
+
+A more comprehensive build guide can be found in the :ref:`build_hdl` user 
+guide.
+
+Resources
+-------------------------------------------------------------------------------
+
+Systems related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Here you can find the quick start guides available for these evaluation boards:
+
+.. list-table::
+   :widths: 20 10 20 20 20 10
+   :header-rows: 1
+
+   * - Evaluation board
+     - Zynq-7000
+     - Zynq UltraScale+ MP
+     - Microblaze
+     - Versal
+     - Arria 10
+   * - AD9434-FMC
+     - :dokuwiki:`ZC706 <resources/fpga/xilinx/fmc/ad9434>`
+     - ---
+     - ---
+     - ---
+     - ---
+
+Hardware related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Product datasheets:
+
+   - :adi:`AD9434`
```

Comment:
One last change, and all looks ok from my side to merge: there is one extra space (indentation should be 2 spaces and not 3) on lines 180 and 181 that are making the list to be generated like a quote or something of the sort; it colors the background.
![image](https://github.com/analogdevicesinc/hdl/assets/54354100/5b3a33fb-4932-40af-aad4-3ee4e6d23a7e)

Reply:
![image](https://github.com/analogdevicesinc/hdl/assets/2892061/ee05cc5f-58d8-41f9-8ef9-37d525257ce8)

There is a confusion on how the lists work, and I agree it's confusing.
Child items must be aligned with the first letter of the parent item, that means, for unordered list `-`/`*`  2 spaces and ordered lists `#.`  3 spaces.